src/app/pageParts/pokemon/index.tsx

```
@@ -9,33 +9,33 @@ import { GetPokemonDataUnionSpeciesListType } from "@/app/type/pokemon";
-  const isBottom = useScrollDetection();
+	const { pokemonData, loaderGetPokemon, isLoading } = useModel({
+		InitialPokemonData,
+	});
+	const isBottom = useScrollDetection();
 
-  useEffect(() => {
-    if (isBottom) {
-      loaderGetPokemon();
-    }
-  }, [isBottom]);
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loaderGetPokemon

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)